Integrations/python/deephaven/pandas/pandas_as_table.py

```diff
@@ -0,0 +1,18 @@
+from deephaven.python_to_java import dataFrameToTable
+from deephaven.plugin.object import Exporter, ObjectType
+from pandas import DataFrame
```

Comment:
We have a choice - this pandas plugin *could* be completely external (like how the matplotlib plugin lives today). The bad part about that right now is that it has knowledge about "deephaven.python_to_java", and potentially other aspects of Integrations. That said, if we are trying to move to be better "python as a deephaven library", we *should* be trying to move towards publishing the python integration points as PyPi packages.